Makefile

```diff
@@ -31,6 +31,16 @@ test:
 docs:
 	tox -e docs
 
+PYTHON3 := $(shell command -v python3 2> /dev/null)
+# target: serve-docs - use a tiny HTTP static server for browsing the doc
+.PHONY: serve-docs
+serve-docs:
```

Comment:
I would have added `docs` as a prerequisite here. What do you think?

Reply:
it slows things a bit, each time you want to serve the docs... I tend to run my HTTP server in one shell session, and the build to be executed from time to time, when I'm ready, asynchronously.

Reply:
OK. 😉